app/_src/gateway-operator/get-started/kic/create-gateway.md

```diff
@@ -70,15 +201,12 @@ spec:
   listeners:
   - name: http
     protocol: HTTP
-    port: 80
-
-' | kubectl apply -f -
+    port: 80' | kubectl apply -f -
 ```
 
 The results should look like this:
 
 ```text
-gatewayconfiguration.gateway-operator.konghq.com/kong created
 gatewayclass.gateway.networking.k8s.io/kong created
```

Comment:
For lines 214-218, I ran that and it worked, but I don't understand why I needed that or what I'm doing/using the variable `PROXY_IP` for.

Reply:
Good call out. I've moved this to the "Add a route" page